gov_uk_dashboards/components/plotly/tooltip.py

```diff
@@ -0,0 +1,32 @@
+"""tooltip"""
+from dash import html
+import dash_bootstrap_components as dbc
+
+
+def tooltip(tooltip_text: str, tooltip_id: str, color: str, font_size: int):
```

Comment:
For the color and font_size, it might be helpful to specify some sensible defaults? Both so developers don't have to add them every time, and so they know what sort of values are reasonable.

Reply:
Good spot, thanks.